libpip2pi/commands.py

```diff
@@ -39,7 +39,11 @@ def try_int(x):
 
 
 if pip_version:
-    if pip_version >= (10, 0, 0):
+    if pip_version >= (19, 3, 1):
+        from pip._internal.main import main as pip_main
```

Comment:
Just a thought, that as [pip user guide](https://pip.pypa.io/en/stable/user_guide/#using-pip-from-your-program) suggests to use pip program and the pip program is defined in entry points in setup.py, would getting that entry point to be a version independent way to find main function. Could something like this work:
```suggestion
        pip_main = pkg_resources.load_entry_point('pip', 'console_scripts', 'pip')
```